### ISSUE TITLE
Username is sporadically cut off

### ISSUE THREAD

**Issue by frdmn** [frdmn/PIA#1]

**Comment by themightyoarfish** [comment]
Did you discover the root of this error by any chance?
This script in working condition would be quite useful.

**Comment by frdmn** [comment]
@themightyoarfish No, unfortunately not. I haven't really used the script anymore so thats why I didn't investigate further. Hopefully I can find some free time to mess around with this soon.
